tests/func/test_remote.py

```diff
@@ -170,6 +170,43 @@ def test(self):
         self.assertEqual(ret, 0)
 
 
+class TestRemoteChecksumJobs(TestDvc):
```

Comment:
@JIoJIaJIu , sorry for the confusion and extra work, but we are in the middle of a migration from `unittest` to `pytest`.
Could you rewrite your tests to `pytest`? Happy to help with further questions about this)

Reply:
> Could you rewrite your tests to pytest?

Have I got right your request?
https://github.com/JIoJIaJIu/dvc/commit/42c34d7269c24187a09bbebe0469647862a75fdd#

Reply:
@JIoJIaJIu, you did) Thanks!